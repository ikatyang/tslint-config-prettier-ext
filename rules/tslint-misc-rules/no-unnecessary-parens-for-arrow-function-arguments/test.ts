const log = (x) => console.log(x);
