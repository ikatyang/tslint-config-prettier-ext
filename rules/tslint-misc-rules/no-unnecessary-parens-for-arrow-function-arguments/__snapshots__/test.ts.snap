// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
ERROR: (no-unnecessary-parens-for-arrow-function-arguments) /rules/tslint-misc-rules/no-unnecessary-parens-for-arrow-function-arguments/test.ts[1, 13]: arrow functions with one argument should not have parentheses around the argument

======
no error

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
const log = (x) => console.log(x);

======
const log = x => console.log(x);

>>>>>> after
"
`;
